### ISSUE TITLE
Is the attribute background useless?

### ISSUE THREAD

**Issue by ZhangChengLin** [Gregwar/Captcha#91]
https://github.com/Gregwar/Captcha/blob/a84b5b450d89fc89aebcfb06b4bbc421c12b797f/src/Gregwar/Captcha/CaptchaBuilder.php#L430

**Is the background useless? It is still related to distort in line 480, because $bg in line 480 does not necessarily work.
I am not quite sure, so I dare not submit a PR rashly, please confirm.**

https://github.com/Gregwar/Captcha/blob/a84b5b450d89fc89aebcfb06b4bbc421c12b797f/src/Gregwar/Captcha/CaptchaBuilder.php#L480

**Comment by visavi** [comment]
Yes, there is a problem with this, but you can turn off all the effects, then it will work
` 
$builder->setBackgroundImages($backgrounds);
$builder->setIgnoreAllEffects(true);
`

**Comment by bytestream** [comment]
Creation of dynamic property removed in https://github.com/Gregwar/Captcha/issues/109
